views/index.erb

```diff
@@ -0,0 +1,5 @@
+<h1> Welcome to Chitter </h1>
+
+<form action = "/peeps">
+  <input type="submit" value="View peeps">
```

Comment:
I like this feature but I don't see a test, was this TDD'd? Also does this work with the form method being stated?